webapp/sources/rudder/rudder-web/src/main/style/rudder/rudder-node.scss

```diff
@@ -103,33 +106,50 @@ $node-state-size: 14px;
 }
 
 .node-state{
-  display: inline-block;
-  width: $node-state-size;
-  height: $node-state-size;
-  border-radius: 50%;
+  display: inline-flex;
+  padding: 6px 8px;
+  border-radius: 4px;
   background-color: #4eb934;
+  color: #fff;
+  font-size: 0.8rem;
+  align-items: baseline;
 
-  &.ignored:after{
+  &:after{
+    content : "ENABLED";
+  }
+  &.ignored{
     background-color: #a4aebf;
+    &:after{
+      content : "IGNORED";
+    }
   }
-  &.empty-policies:after{
+  &.empty-policies{
     background-color: #7591c599;
```

Comment:
Isn't a typo on the color here ? I think it's `#7591c5` ?

Reply:
It's not a typo, Hexa he last two digits represent the alpha channel. So in this case, 99 corresponds to 60% opacity!